code/modules/mob/living/living.dm

```diff
@@ -162,6 +162,8 @@ Sorry Giacom. Please don't be mad :(
 
 	if(AM.Adjacent(src))
 		src.start_pulling(AM)
+	if(pulling == AM)
+		stop_pulling()

```

Comment:
Won't this make pulling impossible? pullin == AM will always be true if the start_pulling(AM) just above succeeds so we immediately stop pulling the thing we started to pull.